config/locales/cy.yml

```diff
@@ -94,10 +94,10 @@ cy:
       description_minimum_html: Gwneir cyfraniadau ar eich enillion cymwys%{tooltip_html} o %{eligible_salary} y flwyddyn.
       description_full_html: Gwneir cyfraniadau ar eich cyflog o %{eligible_salary} y flwyddyn.
       your_contribution_title: Nodwch eich cyfraniad
-      your_contribution_tip: Yr isafswm cyfreithiol yw 1%
+      your_contribution_tip: Yr isafswm cyfreithiol yw 3%
       your_contribution_tip_lt6032: Ar eich lefel cyflog, nid oes isafswm cyfreithiol o gyfraniad ond efallai y bydd eich cynllun pensiwn gweithlu wedi gosod isafswm. Holwch eich cyflogwr.
       employer_contribution_title: Nodwch gyfraniad eich cyflogwr
-      employer_contribution_tip: Yr isafswm cyfreithiol yw 1%
+      employer_contribution_tip: Yr isafswm cyfreithiol yw 2%
```

Comment:
Is it possible to use a variable for the percentage value for this line and line 97?

Reply:
Should be straightforward enough yep.